app/models/supported_ssg.rb

```diff
@@ -9,6 +9,14 @@
   self::SUPPORTED_FILE = Rails.root.join('config/supported_ssg.yaml')
 
   class << self
+    def supported?(ssg_version:, os_version:)
+      ssg_version == ssg_for_os(os_version)
+    end
+
+    def ssg_for_os(os_version)
+      raw_supported.dig('supported', "RHEL-#{os_version}", 'version')
```

Comment:
Observation: So far there was no expectation on the key to include "RHEL-".

Reply:
Not sure I understand; The keys in the config are of the format `RHEL-X.Y`, as agreed upon with the OpenSCAP team. I specifically wanted to be able to look up the supported SSG by OS; that's why it's a dictionary with OS versions as keys and not an array.

Reply:
Sorry, I've meant it more as a note (or something to think about) rather than a concern. Food for thought: There are other ways how to retrieve the information out of already parsed/processed values (`all`).

Reply:
`all` is useless for lookup by OS (but useful for listing all versions, of course). You have to search the array to find what you're looking for. Again, the whole reason we defined the yaml with OS versions as keys to an object is so we can do lookups by OS version.